### ISSUE TITLE
Improve ExecutionHistory

### ISSUE THREAD

**Issue by franzlst** [DLR-RM/RAFCON#124]
- [x] The `ExecutionHistory` should be visualized as a tree. Solving this issue probably also solves #83.
  
  What is internally named a `CallItem` is the parent of all successive Items until the corresponding `ReturnItem` appears:

```
| A - Call (Hierarchy)
| \ 
| | B - Call (Hierarchy)
| | \
| | | C - Call (Execution)
| | | C - Return
| | | D - Call (Execution)
| | | D - Return
| | /
| | B - Return
| | E (Concurrency)
| | \ 
| | | F - Call (Execution)
| | | F - Return
| | | G - Call (Execution)
| | | G - Return
| | /
| | E - Return
| /
| A - Return
```
- [x] The data shown in the context menu of `ReturnItem`s should also list the final outcome of the state and the values of the output data ports instead that of the input data ports

Originally created by @franzlst  (Franz.Steinmetz@dlr.de) at 2016-05-04 15:05:55+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
So the issue  #83 becomes voted up to a bug. :)


Originally created by @Rbelder  at 2016-05-04 18:23:10+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
Please guys, use the "bug" label with care. This is not a mean to express some personal interest to an issue. Although this issue is probably a design improvement it's definitely not a bug. Nor is issue #83.  


Originally created by @sebastian-brunner  (Sebastian.Brunner@dlr.de) at 2016-05-09 05:52:56+00:00 (moved from RMC internal repository)

**Comment by franzlst** [comment]
Fixed with commit 99472f11bd8e51


Originally created by @sebastian-brunner  (Sebastian.Brunner@dlr.de) at 2016-05-13 09:08:00+00:00 (moved from RMC internal repository)
